recipes-multimedia/alsa/alsa-utils_%.bbappend

```diff
@@ -0,0 +1,21 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+
+OVERRIDES_append = "${@bb.utils.contains("PACKAGECONFIG", "udev", ":udev-enabled", "",d)}"
+
+SRC_URI_append_hardkernel-odroidn2plus_udev-enabled = " file://soundconfig \
+                                                        file://90-alsa-restore.rules \
+                                                      "
+
+FILES_alsa-utils-alsactl_append_hardkernel-odroidn2plus_udev-enabled = " ${libdir}/udev/soundconfig \
+                                                                        "
+
+do_install_append_hardkernel-odroidn2plus_udev-enabled() {
+    #We need to remove default rules file
+    rm -rf ${D}/lib/udev/rules.d/90-alsa-restore.rules ${D}/${libdir}/udev/rules.d/90-alsa-restore.rules
+    rm -d ${D}/lib/udev/rules.d && rm -d ${D}/lib/udev && rm -d ${D}/lib || :
+    install -d ${D}/${libdir}/udev/rules.d/
+    install "${B}/../soundconfig" ${D}/${libdir}/udev
+    install "${B}/../90-alsa-restore.rules" ${D}/${libdir}/udev/rules.d
```

Comment:
install -m 0644 ${WORKDIR}/90-alsa-restore.rules ${D}`pkg-config --variable=udevdir udev`/rules.d/90-alsa-restore.rules

I need to check this one, there is some tweak with "unwanted" dir set as udev rules dir in poky's alsa-utils. It doesn't exist after regular do_install